Hardcore.lua

```diff
@@ -2065,6 +2019,103 @@ function Hardcore:SetGriefAlertCondition(grief_alert_option)
 	end
 end
 
+function Hardcore:DKConvert(dk_convert_option) 
+	local _, _, classID = UnitClass("player")
+	local level = UnitLevel("player")
+	local inCombat = UnitAffectingCombat("player")
+	if dk_convert_option == "sacrifice" then
+		if inCombat == true then
+			Hardcore:Print("Can't use sacrifice in combat")
+			return
+		end
+		-- check if eligible
+		if classID == 6 then
+			Hardcore:Print("You can't sacrifice Death Knight character")
+			return
+		end
+		if level ~= SACRIFICE_LEVEL then
+			Hardcore:Print(string.format("You must be level %s to sacrifice", SACRIFICE_LEVEL))
+			return
+		end
+		-- need to warn before sacrifice if something is wrong
+		local debug_message = "Playtime gap percentage: " .. Hardcore_Character.tracked_played_percentage .. "%."
+		Hardcore:Debug(debug_message)
+		local percentage = Hardcore_Character.tracked_played_percentage
+
+		if Hardcore:ShouldShowPlaytimeWarning(level, percentage) then
+			Hardcore:DisplayPlaytimeWarning(level)
+			return
+		end
+		-- we deaths, bh, played_time_gaps and trade partners
+		if (#Hardcore_Character.deaths > 0 or #Hardcore_Character.bubble_hearth_incidents > 0 or #Hardcore_Character.played_time_gap_warnings > 0 or #Hardcore_Character.trade_partners > 0) then
```

Comment:
I think the player gap warnings should be removed from the conditional here

Reply:
True. We discussed that and I just forgot it.